pyCRLD/Environments/HeterogeneousObservationsEnv.py

```diff
@@ -0,0 +1,170 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: ../../nbs/Environments/02_HeterogeneousObservationsEnv.ipynb.
```

Comment:
Most of the changes are within this file. It's largely adapting the `ebase` file to deal with multiple observations.